src/System.Net.Sockets/tests/FunctionalTests/SocketOptionNameTest.cs

```diff
@@ -159,6 +159,28 @@ public void MulticastInterface_Set_InvalidIndex_Throws()
             }
         }
 
+        [OuterLoop] // TODO: Issue #11345
+        [Fact]
+        public void MulticastInterface_Set_WithIPv6()
+        {
+            using (Socket s = new Socket(AddressFamily.InterNetworkV6, SocketType.Dgram, ProtocolType.Udp))
+            {
+                s.SetSocketOption(SocketOptionLevel.IPv6, SocketOptionName.MulticastInterface, 1);
+            }
+        }
+
+        [OuterLoop] // TODO: Issue #11345
+        [Fact]
+        public void MulticastInterface_Set_InvalidIndexWithIPv6_Throws()
+        {
+            int interfaceIndex = 31415;
+            using (Socket s = new Socket(AddressFamily.InterNetworkV6, SocketType.Dgram, ProtocolType.Udp))
+            {
+                Assert.Throws<SocketException>(() =>
+                                               s.SetSocketOption(SocketOptionLevel.IPv6, SocketOptionName.MulticastInterface, interfaceIndex);
```

Comment:
This doesn't compile.  It's missing a `)`.

Reply:
Yes I noticed this, I fixing it and will push with some extra fixes requires in pal_networking.c